core/store/src/test/java/uk/gov/gchq/gaffer/store/operation/handler/compare/SortHandlerTest.java

```diff
@@ -33,44 +32,31 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.Random;
-import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class SortHandlerTest {
 
     @Test
-    public void shouldSortBasedOnProperty() throws OperationException, JsonProcessingException {
+    public void shouldSortBasedOnProperty() throws OperationException {
         // Given
-        final Entity entity1 = new Entity.Builder().group(TestGroups.ENTITY)
-                .property("property", 1)
-                .build();
-        final Entity entity2 = new Entity.Builder().group(TestGroups.ENTITY)
-                .property("property", 2)
-                .build();
-        final Entity entity3a = new Entity.Builder().group(TestGroups.ENTITY)
-                .property("property", 3)
-                .property("otherProp", "a")
-                .build();
-        final Entity entity3b = new Entity.Builder().group(TestGroups.ENTITY)
-                .property("property", 3)
-                .property("otherProp", "b")
-                .build();
-        final Entity entity4 = new Entity.Builder().group(TestGroups.ENTITY)
-                .property("property", 4)
-                .build();
+        final Entity entity1 = makeEntity(1);
+        final Entity entity2 = makeEntity(2);
+        final Entity entity3a = makeEntity(3, "a");
+        final Entity entity3b = makeEntity(3, "b");
+        final Entity entity4 = makeEntity(4);
 
         final List<Entity> input = Lists.newArrayList(entity1, entity4, entity3a, entity3b, entity2);
 
         final Sort sort = new Sort.Builder()
                 .input(input)
                 .comparators(new ElementPropertyComparator.Builder()
                         .groups(TestGroups.ENTITY)
-                        .property("property")
+                        .property("property1")
```

Comment:
Hi - there wasn't any major reason, it was just part of refactoring Builder() methods for all the Objects that had one or two properties in that test class - so thought it might be more readable or imply that more than one property can be added to the Builder() in this instance.

I can revert the change you prefer and if it read better before?